src/js/utilities/ajax.js

```diff
@@ -184,7 +190,8 @@ Crocodoc.addUtility('ajax', function (framework) {
             var opt = parseOptions(options),
                 method = opt.method,
                 data = opt.data,
-                headers = opt.headers;
+                headers = opt.headers,
+                withCredentials = !!opt.withCredentials;
```

Comment:
Another thing I just noticed, maybe parseOptions (few lines above) should be changed to add withCredentials.
options.withCredentials = !!options.withCredentials;
Makes a wee bit more consistent.